File: app/src/main/java/com/rkbapps/neetflix/models/tvseries/TvSeriesResult.java
package com.rkbapps.neetflix.models.tvseries;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class TvSeriesResult {
    @SerializedName("first_air_date")
    @Expose
    private String firstAirDate;
    @SerializedName("id")
    @Expose
    private Integer id;
    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("poster_path")
    @Expose
    private String posterPath;
    @SerializedName("vote_average")
    @Expose
    private Integer voteAverage;

    public TvSeriesResult() {
    }

    public String getFirstAirDate() {
        return firstAirDate;
    }

    public Integer getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getPosterPath() {
        return posterPath;
    }

    public void setPosterPath(String posterPath) {
        this.posterPath = posterPath;
    }

    public Integer getVoteAverage() {
        return voteAverage;
    }

    public void setVoteAverage(Integer voteAverage) {
        this.voteAverage = voteAverage;
    }
}

File: app/src/main/java/com/rkbapps/neetflix/models/tvseries/TvSeriesResult.kt
package com.rkbapps.neetflix.models.tvseries

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class TvSeriesResult(
    @SerializedName("first_air_date")
    @Expose
    val firstAirDate: String,

    @SerializedName("id")
    @Expose
    val id: Int,

    @SerializedName("name")
    @Expose
    val name: String,

    @SerializedName("poster_path")
    @Expose
    var posterPath: String?,

    @SerializedName("vote_average")
    @Expose
    var voteAverage: Double,

    )